### ISSUE TITLE
MIS Issue

### ISSUE THREAD

**Issue by karolserkis** [PeaWagon/Kaplan#37]
Hello @PeaWagon 

I apologize for the lateness of my issues. Perhaps they can be useful for the final submission at this point.

9.4.4 Access Routine Semantics: get fitness(xyz coords, method, basis, fit form, coef energy, coef rmsd, charge, multip): why is the transition: None? and exception: None? 
Shouldn't there be at least be an exception as calc fitness has?

[MIS.pdf]
(https://github.com/PeaWagon/Kaplan/blob/master/docs/Design/MIS/MIS.pdf)

**Comment by PeaWagon** [comment]
Thanks for the comments, I appreciate them. I have actually changed how the fitg module works. I realised that making get_fitness as a function was completely redundant, so I made calc_fitness, sum_energies, and sum_rmsds the access functions to import for this module. I will re-write the specification. The exception that calc_fitness raises should not need to be present (since the fit_form value is checked in the ga input module, it is redundant to check it again here). The fitness module doesn't actually do much except determine how energy and rmsd values are combined to give a fitness value for a population member (pmem). The ring is responsible for updating a pmem's fitness (so therefore no transition).
